### ISSUE TITLE
Worker Hours incomplete

### ISSUE THREAD

**Issue by arroyoDev** [CrisisCleanup/crisiscleanup-3-web#200]
When reporting hours, the behavior should add rows to a table, not add to a single line.

Current:  
<img width="367" alt="image" src="https://user-images.githubusercontent.com/699333/74647387-2d1fbc00-5141-11ea-8af9-fc8095cbea67.png">

This one was really hard to find in invision, but here is the link to how the table should look:
![image](https://user-images.githubusercontent.com/699333/74647447-41fc4f80-5141-11ea-8714-2f30d77bcdcf.png)

https://projects.invisionapp.com/d/main#/console/14746784/314349326/preview

I think the table pretty much shows how it should behave.  Note when I typed in hours, the grey text placeholders of "volunteers" and "hours per volunteer" did not go away, but remained in the box and pushed my text halfway below the box.  

In short, I think the top is simply an entry spot to populate the table below.  if there a better way... make it so...

**Comment by aarontitus** [comment]
Approved. I'm getting weird 500 errors, but I think they're unrelated.
